sentry-spring-boot-starter/src/main/java/io/sentry/spring/boot/ContextTagsEventProcessor.java

```diff
@@ -0,0 +1,41 @@
+package io.sentry.spring.boot;
+
+import io.sentry.EventProcessor;
+import io.sentry.SentryEvent;
+import io.sentry.SentryOptions;
+import io.sentry.util.CollectionUtils;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.slf4j.MDC;
+
+/**
+ * Attaches context tags defined in {@link SentryOptions#getContextTags()} from {@link MDC} to
+ * {@link SentryEvent#getTags()}.
+ */
+public final class ContextTagsEventProcessor implements EventProcessor {
```

Comment:
Should this event processor live in the `sentry-spring` module?

Reply:
It put it to `sentry-spring-boot-starter` because it needs `slf4j` and it is provided only in that module. If we decide to move it somewhere else and add there `sl4fj` perhaps it could be `sentry` module as there is nothing really Spring specific here.